src/Goerli-DssSpell.t.sol

```diff
@@ -511,530 +452,29 @@ contract DssSpellTest is GoerliDssSpellTestBase {
-        // expression above, we need to divide the price by WAD.
-        //
-        //     gem*WAD       dai*WAD      /       (dai/gem)*WAD            / WAD
-        //        v             v                       v                     v
-        //    1 * WAD - (1_000_000 * WAD) / (30_043_520_665599336150000000 / WAD)
-        //
-        // While mathematically correct, the expression will present a rounding error with Solidity's integer math, so
-        // we reorder it as follows:
-        uint256 skimmedInk = (1_000_000 * WAD) * WAD / 30_043_520_665599336150000000;
-        assertEq(ink, 1 * WAD - skimmedInk, "RWA008: wrong ink in urn after skim");
-        assertEq(art, 0, "RWA008: wrong art in urn after skim");
-
-        hevm.warp(block.timestamp + end.wait());
-
-        vow.heal(min(vat.dai(address(vow)), sub(sub(vat.sin(address(vow)), vow.Sin()), vow.Ash())));
-
-        // Removing the surplus to allow continuing the execution.
-        hevm.store(
-            address(vat),
-            keccak256(abi.encode(address(vow), uint256(5))),
-            bytes32(uint256(0))
-        );
-
-        end.thaw();
-
-        end.flow("RWA008-A");
-
-        dai.approve(address(daiJoin), 1_000_000 * WAD);
-        daiJoin.join(address(this), 1_000_000 * WAD);
-
-        vat.hope(address(end));
-        end.pack(1_000_000 * WAD);
-
-        assertEq(vat.gem("RWA008-A", address(this)), 0, "RWA008: wrong vat gem");
-        assertEq(rwagem_008.balanceOf(address(this)), 0, "RWA008: wrong gem balance");
-        end.cash("RWA008-A", 1_000_000 * WAD);
-        assertGt(vat.gem("RWA008-A", address(this)), 0, "RWA008: wrong vat gem after cash");
-        assertEq(rwagem_008.balanceOf(address(this)), 0, "RWA008: wrong gem balance after cash");
-        rwajoin_008.exit(address(this), vat.gem("RWA008-A", address(this)));
-        assertEq(vat.gem("RWA008-A", address(this)), 0, "RWA008: wrong vat gem after exit");
-        assertGt(rwagem_008.balanceOf(address(this)), 0, "RWA008: wrong gem balance after exit");
-    }
-
-    function testRWA009_INTEGRATION_BUMP() public {
-        vote(address(spell));
-        scheduleWaitAndCast(address(spell));
-        assertTrue(spell.done());
-
-        giveAuth(address(oracle), address(this));
-
-        (, address pip, , ) = oracle.ilks("RWA009-A");
-
-        assertEq(DSValueAbstract(pip).read(), bytes32(100 * MILLION * WAD), "RWA009: Bad initial PIP value");
-
-        oracle.bump("RWA009-A", 110 * MILLION * WAD);
-
-        assertEq(DSValueAbstract(pip).read(), bytes32(110 * MILLION * WAD), "RWA009: Bad PIP value after bump()");
-    }
-
-    function testRWA009_INTEGRATION_TELL() public {
-        vote(address(spell));
-        scheduleWaitAndCast(address(spell));
-        assertTrue(spell.done());
+    function testRWA009_SPELL_DRAW() public {
+        address rwaUrn009       = addr.addr("RWA009_A_URN");
+        address rwaUrn009Output = addr.addr("MCD_PAUSE_PROXY"); // for goerli, we use the pause proxy
```

Comment:
```suggestion
        address rwaUrn009Output = addr.addr("RWA009_A_OUTPUT_CONDUIT"); // for goerli, we use the pause proxy
```

this is what I was telling you

Reply:
oh i didn't see that they added the key too, thanks!